### ISSUE TITLE
Battery and pairing status icons

### ISSUE THREAD

**Issue by flavio** [JanBobolz/agendwatchfacePebble#3]
I really really love this watchface. I think it's almost perfect.

I would love it even more if you added:
- A battery indicator showing the level of charge of the watch.
- A bluetooth icon when the watch is paired with the mobile phone. The icon should disappear/change when there's no connection.

Thanks for your help and kudos for publishing the code!

**Comment by JanBobolz** [comment]
Hey, thanks!
- A textual battery indicator appears when there are only 20 % or less left. It's where usually the current weekday would show up. I figured that would be enough of a "you should charge at some point" warning. 
- The sync indicator does this reasonably well. While syncing, there is a bar at the very top of the watchface that disappears when done. When there is no connection, the bar is there permanently, basically saying that the sync does not progress, which should be equivalent to saying that the watch is not connected.

So I'm inclined to close this issue :) - do you agree?

**Comment by flavio** [comment]
I recently noticed the battery indicator. I think this works pretty well.

As for the sync indicator, this is really hard to notice because the bar is really thin. Now I am able to spot it because of your message. I think an icon would provide a better user experience, but feel free to close this issue. At least now I know what to look for.

Cheers

**Comment by aeroechelon** [comment]
Just chiming in on this particular watchface as it's one of my favourite watchfaces as well. I love the extensibility and the amount of work put into creating this application.

Personally, one of the things I dislike about a majority of Pebble apps is knowing of some sort of 'sync' indicator. To me it's a lot of frequent overhead when navigating to / from a watchface that requires syncing. I wish it would display simply the previous information and persist this information across app launches, which this app does well.

In a nutshell, I think the sync indicator is visible enough to provide information for those do care about this information and still subtle for those who want to ignore it.

**Comment by JanBobolz** [comment]
I agree :)
Thanks for the feedback!
